schema.sql

```diff
@@ -11,3 +11,40 @@ PRIMARY KEY(id)
 );
 ALTER TABLE animals
 ADD species VARCHAR(50);
+/************************************************************************/
+CREATE TABLE animals_backup AS SELECT * FROM animals;
+SELECT 10
+DROP TABLE animals;
+DROP TABLE
+CREATE TABLE animals (
+id SERIAL PRIMARY KEY,
+name VARCHAR(100),
+date_of_birth DATE,
+escape_attempts INT,
+neutered BOOLEAN,
+weight_kg DECIMAL,
+species VARCHAR(100)
+);
+CREATE TABLE
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+SELECT id, name, date_of_birth, escape_attempts, neutered, weight_kg, species
+FROM animals_backup;
+INSERT 0 10
+SELECT * FROM animals; 
+ALTER TABLE animals
+DROP  COLUMN species;
+ALTER TABLE
+ALTER TABLE animals
+ADD species_id INT,
+ADD CONSTRAINT fk_species_id
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+ALTER TABLE
+ALTER TABLE animals
+ADD owner_id INT,
+ADD CONSTRAINT fk_owner_id
+FOREIGN KEY (owner_id)
+REFERENCES owners(id);
+ALTER TABLE
+
```

Comment:
Nice Job so far👍. Probably you might have forgotten😉. but we are required to update our `schema.sql` file with the commands we used in creating the tables and modifying the animal table for this milestone😉. Kindly do well to add the commands to your `schema.sql` file👍.

<img width="593" alt="image" src="https://github.com/Christelle-12/vet_clinic/assets/64643995/00444e9d-f82f-411f-80e1-13a5e01dac26">